packages/tasks/src/model-libraries.ts

```diff
@@ -394,6 +394,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: { term: { path: "model.safetensors" } },
 		snippets: snippets.stable_audio_tools,
 	},
+	"stable-diffusion-3": {
```

Comment:
There's some ongoing discussion on Slack of whether to do this or not. If we go ahead, we'll need a different name, because otherwise all repos with a tag `stable-diffusion-3` will get this library automatically.